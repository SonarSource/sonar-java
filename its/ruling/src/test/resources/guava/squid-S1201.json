{
'com.google.guava:guava:src/com/google/common/base/Equivalence.java':[
314,
],
}
