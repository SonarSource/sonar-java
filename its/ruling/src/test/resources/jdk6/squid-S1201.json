{
'jdk6:java/awt/font/GlyphVector.java':[
593,
],
'jdk6:java/awt/geom/Area.java':[
482,
],
}
