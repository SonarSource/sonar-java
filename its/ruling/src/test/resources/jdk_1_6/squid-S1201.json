{
'jdk_1_6:java/awt/DisplayMode.java':[
109,
],
'jdk_1_6:java/awt/MenuShortcut.java':[
117,
],
'jdk_1_6:java/awt/datatransfer/DataFlavor.java':[
915,
981,
],
'jdk_1_6:java/awt/font/FontRenderContext.java':[
299,
],
'jdk_1_6:java/awt/font/GlyphVector.java':[
593,
],
'jdk_1_6:java/awt/font/ImageGraphicAttribute.java':[
185,
],
'jdk_1_6:java/awt/font/ShapeGraphicAttribute.java':[
214,
],
'jdk_1_6:java/awt/font/TextHitInfo.java':[
143,
],
'jdk_1_6:java/awt/font/TextLayout.java':[
2578,
],
'jdk_1_6:java/awt/geom/Area.java':[
482,
],
'jdk_1_6:java/beans/DefaultPersistenceDelegate.java':[
199,
],
'jdk_1_6:java/beans/MetaData.java':[
1265,
],
'jdk_1_6:java/net/HttpCookie.java':[
1140,
],
'jdk_1_6:java/net/URLStreamHandler.java':[
313,
],
'jdk_1_6:java/sql/Timestamp.java':[
397,
],
'jdk_1_6:java/text/Collator.java':[
335,
],
'jdk_1_6:java/util/Arrays.java':[
2132,
2161,
2190,
2219,
2248,
2277,
2312,
2347,
2379,
],
}
