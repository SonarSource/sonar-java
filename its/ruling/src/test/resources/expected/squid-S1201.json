{
'project:com/sun/corba/se/impl/io/ValueUtility.java':[
354,
],
'project:com/sun/corba/se/impl/javax/rmi/CORBA/StubDelegateImpl.java':[
109,
],
'project:com/sun/corba/se/impl/protocol/CorbaClientDelegateImpl.java':[
310,
],
'project:com/sun/corba/se/spi/ior/iiop/GIOPVersion.java':[
91,
],
'project:com/sun/imageio/plugins/png/PNGImageWriter.java':[
956,
],
'project:com/sun/jmx/mbeanserver/Introspector.java':[
481,
],
'project:com/sun/jmx/snmp/agent/SnmpIndex.java':[
109,
],
'project:com/sun/mirror/util/SourceOrderDeclScanner.java':[
61,
],
'project:com/sun/org/apache/bcel/internal/classfile/Utility.java':[
1113,
],
'project:com/sun/org/apache/bcel/internal/generic/InstructionComparator.java':[
76,
111,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RegularExpression.java':[
2348,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/identity/IdentityConstraint.java':[
147,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/util/XInt.java':[
45,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/SoftReferenceGrammarPool.java':[
294,
],
'project:com/sun/org/apache/xerces/internal/util/XMLGrammarPoolImpl.java':[
300,
],
'project:com/sun/org/apache/xerces/internal/xni/XMLString.java':[
140,
162,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMNodeProxy.java':[
120,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/ExtendedType.java':[
109,
],
'project:com/sun/org/apache/xml/internal/utils/QName.java':[
588,
],
'project:com/sun/org/apache/xml/internal/utils/XMLString.java':[
140,
],
'project:com/sun/org/apache/xml/internal/utils/XMLStringDefault.java':[
168,
],
'project:com/sun/org/apache/xpath/internal/compiler/Compiler.java':[
295,
],
'project:com/sun/org/apache/xpath/internal/objects/XBoolean.java':[
151,
],
'project:com/sun/org/apache/xpath/internal/objects/XBooleanStatic.java':[
60,
],
'project:com/sun/org/apache/xpath/internal/objects/XNodeSet.java':[
703,
],
'project:com/sun/org/apache/xpath/internal/objects/XNull.java':[
128,
],
'project:com/sun/org/apache/xpath/internal/objects/XNumber.java':[
395,
],
'project:com/sun/org/apache/xpath/internal/objects/XObject.java':[
635,
],
'project:com/sun/org/apache/xpath/internal/objects/XRTreeFrag.java':[
257,
],
'project:com/sun/org/apache/xpath/internal/objects/XString.java':[
293,
337,
],
'project:com/sun/org/apache/xpath/internal/objects/XStringForFSB.java':[
291,
333,
378,
],
'project:java/awt/DisplayMode.java':[
127,
],
'project:java/awt/MenuShortcut.java':[
148,
],
'project:java/awt/datatransfer/DataFlavor.java':[
925,
993,
],
'project:java/awt/font/FontRenderContext.java':[
317,
],
'project:java/awt/font/GlyphVector.java':[
610,
],
'project:java/awt/font/ImageGraphicAttribute.java':[
203,
],
'project:java/awt/font/ShapeGraphicAttribute.java':[
232,
],
'project:java/awt/font/TextHitInfo.java':[
161,
],
'project:java/awt/font/TextLayout.java':[
2596,
],
'project:java/awt/geom/Area.java':[
500,
],
'project:java/lang/invoke/MethodType.java':[
591,
],
'project:java/net/URLStreamHandler.java':[
330,
],
'project:java/nio/ByteBuffer.java':[
1134,
],
'project:java/nio/CharBuffer.java':[
1134,
],
'project:java/nio/DoubleBuffer.java':[
1134,
],
'project:java/nio/FloatBuffer.java':[
1134,
],
'project:java/nio/IntBuffer.java':[
1134,
],
'project:java/nio/LongBuffer.java':[
1134,
],
'project:java/nio/ShortBuffer.java':[
1134,
],
'project:java/sql/Timestamp.java':[
400,
],
'project:java/text/Collator.java':[
352,
],
'project:java/util/Arrays.java':[
1610,
1639,
1668,
1697,
1726,
1755,
1790,
1825,
1856,
],
'project:java/util/EnumMap.java':[
675,
],
'project:java/util/Objects.java':[
56,
],
'project:javax/management/openmbean/CompositeDataInvocationHandler.java':[
230,
],
'project:javax/rmi/CORBA/StubDelegate.java':[
64,
],
'project:javax/swing/plaf/nimbus/ImageCache.java':[
210,
],
'project:org/ietf/jgss/GSSName.java':[
180,
],
'project:org/omg/CORBA/portable/Delegate.java':[
405,
],
}
